visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -35,9 +35,9 @@ export interface ICohortEditorStyles {
     saveAndDeleteDiv: IStyle;
 }
 
-const theme = getTheme();
 
 export const cohortEditorStyles: () => IProcessedStyleSet<ICohortEditorStyles> = () => {
+    const theme = getTheme();
```

Comment:
@janhavi13 I made some changes to this file as well as where it is consumed (Cohort Editor.tsx) to get theming working for the editor. One issue is the call to getTheme needs to happen in the function, otherwise its set at load time and never changes. Similarly in CohortEditor.tsx, the call to get styles needs to happen in render, to ensure its being rendered with the current theme. The second point is to try to use semanticColors where possible (this is more for Fairlearn theming, I can update any points in this code that aren't theming right.). You put a lot of time into this component and it looks great, I'm sorry if the theming was giving you problems, moving the loading of the theme is definitely one of those things that you can try to debug for hours, and I hope it didn't take up too much of your time.

Reply:
@rihorn2 - Thanks for the tip on getTheme! I followed the same pattern for fairlearn and I'm sure i would have spent a lot of time debugging it  :| 
I have been careful to use semanticColors in Fairlearn but sure feel free to update the code if you find any issues :)